Classes/Service/AuthenticationService.php

```diff
@@ -131,9 +131,13 @@ public function getUser()
             // dispatch a signal (containing the user with his access token if auth was successful)
             // so other extensions can use them to make further requests to an API
             // provided by the authentication server
-            /** @var Dispatcher $dispatcher */
-            $dispatcher = GeneralUtility::makeInstance(ObjectManager::class)->get(Dispatcher::class);
-            $dispatcher->dispatch(__CLASS__, 'getUser', [$user]);
+            $versionInformation = GeneralUtility::makeInstance(Typo3Version::class);
```

Comment:
I suggest dropping this old signal completely as a breaking change. (it will be a new major version anyways)

Reply:
dropped